dev/build_release.py

```diff
@@ -516,23 +665,35 @@ def build_packages(self):
       if self.__options.nebula:
         return
 
+      # Do not choke if there is nothing to copy
       wait_on = set(all_subsystems).difference(set(self.__build_failures))
-      pool = multiprocessing.pool.ThreadPool(processes=len(wait_on))
-      print 'Copying packages...'
-      pool.map(self.__do_copy, wait_on)
+      if len(wait_on) > 0:
+        pool = multiprocessing.pool.ThreadPool(processes=len(wait_on))
+        print 'Copying packages...'
+        pool.map(self.__do_copy, wait_on)
+      else:
+        print 'Nothing to copy.'
       return
 
   def __do_copy(self, subsys):
     print 'Starting to copy {0}...'.format(subsys)
-    pids = self.start_copy_debian_target(subsys)
+    if self.__options.platform == 'debian':
+      pids = self.start_copy_debian_target(subsys)
+
+    elif self.__options.platform == 'redhat':
+      pids = self.start_copy_redhat_target(subsys)
 
     for p in pids:
       p.check_wait()
     print 'Finished copying {0}.'.format(subsys)
 
   @classmethod
-  def init_argument_parser(cls, parser):
+  def init_argument_parser(cls, parser, valid_platforms):
```

Comment:
This was not working at some time. Probably because of all my hacks in the previous script. Works now.. Done.